### PR TITLE
Adjust IANA language

### PR DESCRIPTION
**Pull request by randall**
Stuart has registered nq @ https://www.iana.org/assignments/service-names-port-numbers

### CODE REVIEW

File: draft-ietf-ippm-responsiveness.md [hunk 1]
```diff
@@ -854,9 +854,11 @@ TBD
...
 # IANA Considerations
 
-IANA has been requested to record the service type
-“_nq._tcp” (Network Quality)
-for advertising and discovery of Responsiveness Test Server instances.
+IANA has allocated the service type “_nq._tcp” (Network Quality)
+for advertising and discovery of Responsiveness Test Server
+instances in the "Service Name and Transport Protocol Port Number Registry"
+defined by {{RFC6335}}, located at
+<https://www.iana.org/assignments/service-names-port-numbers>.
```
Comment (hawkinsw) [inline draft-ietf-ippm-responsiveness.md:861]: I assume that this verbiage is best practice, but from what we got from IANA via email, I cannot seem to find the specifics.

**Review by hawkinsw — COMMENTED** [review]
I hope my little feedback is helpful!

File: draft-ietf-ippm-responsiveness.md (hunk 1 above)
Comment (randall) [reply draft-ietf-ippm-responsiveness.md:861]: I ripped that bit from another RFC.

are you saying we should just drop it? Or is there other wording we should use?

File: draft-ietf-ippm-responsiveness.md [hunk 2]
```diff
@@ -854,9 +854,11 @@ TBD
 
 # IANA Considerations
 
-IANA has been requested to record the service type
-“_nq._tcp” (Network Quality)
-for advertising and discovery of Responsiveness Test Server instances.
+IANA has allocated the service type “_nq._tcp” (Network Quality)
```
[the remark this answers is not in the document]
Comment (randall) [reply draft-ietf-ippm-responsiveness.md]: I killed all of them.

**Pull request merged**